backend/tests/test_participant_details.py

```diff
@@ -60,3 +60,20 @@ def test_get_delete_put_participant_unauthorized(client, add_participants, metho
     response = rv.get_json()
     assert rv.status_code == HTTPStatus.UNAUTHORIZED
     assert response["msg"] == "Missing Authorization Header"
+
+@pytest.mark.parametrize("field", ["email", "github"])
+def test_try_edit_participant_with_existing_email(auth_client, add_participants, field):
+    """Test try to edit participant with other participant email address."""
```

Comment:
Docstring points that it's only test for email. ;-)

No newline at the end of the file.